bot/main.py

```diff
@@ -186,6 +186,22 @@ async def roll_error(ctx, error):
         await ctx.reply("This server is not for academic purposes.")
 
 
+@bot.event
+async def on_guild_join(guild):
+    server_id = str(guild.id)
+    server_config = get_config(server_id)
+
+    welcome_message = "CAS-bot has joined this server"
+
+    if server_config is None:
+        welcome_message = "Read the instructions to invite the bot in the project README"
```

Comment:
Hmmm, I don't think we need this function. 

This function will only be called once when the bot is added to this server. But this error message must be printed everytime someone tries to do .verify